Hello, World! BRAPH 2 Distribution

iok-ui opened the pull request:
This PR introduces `get_braph2distribution`, a function for generally generating a BRAPH 2 distribution that includes only the elements an user specifies. The resulting output is a **standalone**, **GUI-ready**, and **fully unit-tested** distribution of software—all while ensuring it retains the essential BRAPH-2 functionality an end user needs.

This PR illustrates the usage of `get_braph2distribution` through **Hello, World!** example:

## **Hello, World!**

- Demonstrates a simple pipeline where the user enters the first string (e.g., “Hello”) and the second string (e.g., “World”), and the pipeline combines them into a single string (“Hello World!”).

- Showcases the minimal overhead needed to create a fully compiled, self-contained application with a basic GUI and passing the unit tests (launcher function also changed to `braph2helloworld`).

<img width="976" alt="image" src="https://github.com/user-attachments/assets/3c798180-6199-4f39-a374-503199943046" />

<img width="641" alt="image" src="https://github.com/user-attachments/assets/62fb969c-55ff-4bd7-be38-73044c6cb8b6" />


<img width="993" alt="image" src="https://github.com/user-attachments/assets/37d482e4-4f38-436c-a886-2e6c1e220d4b" />

Commit:
u

Commit:
u

Commit:
Update BRAPH2GUI.m

Commit:
u

Commit:
u

Commit:
u

Commit:
Update test_braph2.m

Commit:
u

Commit:
u

Commit:
Merge branch 'develop' into ywc-lite-genesis

Commit:
Update get_braph2distribution.m

Commit:
Merge branch 'ywc-lite-genesis' of https://github.com/braph-software/BRAPH-2 into ywc-lite-genesis

giovannivolpe committed:
Gv lite genesis (#1790)

Co-authored-by: Yu-Wei Chang <hejyuwei@gmail.com>

giovannivolpe committed:
Update BRAPH2.m

giovannivolpe committed:
Update braph2rollcall_config.m

giovannivolpe committed:
Update braph2genesis.m

giovannivolpe committed:
Update braph2rollcall_config.m

giovannivolpe committed:
Update braph2rollcall_config.m

Commit:
Update braph2genesis.m

Commit:
rm braph2helloworld

c-yuwei commented:
4 testing cases:

Distr 1 - `braph2genesis('braph2genesis/sandbox/braph2rollcall_config.m')`: Tested
Distr 2 - `braph2genesis()`: Testing
Distr 3 - `braph2genesis('braph2helloworld_config.m')`: Tesetd
Distr 4 - `braph2genesis('braph2memorycapacity_config.m')`: Tesetd

Commit:
revert back to previou version

Commit:
braph2genesis function finalization (#1795)

Commit:
Update for braph2 the folder (#1792)

bzufiriaki committed:
BRAPH 2 Hello, World! Distribution Tutorial (#1793)

Co-authored-by: Yu-Wei Chang <hejyuwei@gmail.com>

Commit:
Update README.md (#1796)

giovannivolpe committed:
readmes

giovannivolpe committed:
Update Didtribution Docs

Create readme.md

Update readme.md

Update dev_distribution.tex

Update dev_distribution.pdf

Update readme.md

Commit:
Update for release date and the layout of landing GUI (#1797)

bzufiriaki committed:
Modified tutorial according to comments (#1798)
